app/serializers/api/v1/historical_emissions/record_serializer.rb

```diff
@@ -3,6 +3,7 @@ module V1
     module HistoricalEmissions
       class RecordSerializer < ActiveModel::Serializer
         belongs_to :location
+        belongs_to :iso_code3
```

Comment:
iso_code3 is not a relation but an attribute, so you should replace `belongs_to` with `attribute`.

it would perhaps make sense to show the iso code and location names under a `location` key.

Reply:
I've changed to attribute, and I agree but let's put both inside the location key later as it means some other changes.

Thanks Mario!